packages/react-dev-overlay/src/middleware.ts

```diff
@@ -67,6 +67,99 @@ function getSourcePath(source: string) {
   return source
 }
 
+async function findOriginalSourcePositionAndContent(
+  webpackSource: any,
+  position: { line: number; column: number | null }
+) {
+  const consumer = await new SourceMapConsumer(webpackSource.map())
+  const sourcePosition: NullableMappedPosition = consumer.originalPositionFor({
+    line: position.line,
+    column: position.column ?? 0,
+  })
+
+  if (!sourcePosition.source) {
+    return null
```

Comment:
This leaks the consumer, missing:
```ts
consumer.destroy()
```

Maybe we should wrap this in a `try { } finally { }`?